core/lib/i18n/README.md

```diff
@@ -211,6 +211,23 @@ CTC is a name that is distinct and identifies this as the Chrome translation for
 }
 ```
 
+### Collisions
```

Comment:
history has shown we pretty reliably forget this information over time, so please give any feedback on ambiguity so we can clear it up now :)